sample/sample-with-di/app/src/main/java/com/dropbox/kaiken/sample/advancedsample/app/AdvancedKaikenSampleApplication.kt

```diff
@@ -68,3 +72,24 @@ interface SkeletonComponent : SdkSpec {
         ): SkeletonComponent
     }
 }
+
+@Module
+@ContributesTo(AppScope::class)
+class BindingModule {
+    @Provides
+    @SingleIn(AppScope::class)
+    fun provideUserFlow(): @JvmSuppressWildcards MutableSharedFlow<DiSampleUser> =
+        MutableSharedFlow(replay = 1)
```

Comment:
I was never sure if this should be 1 or 0, also maybe it is a state flow now that it is always all user set

Reply:
Yup, stateflow makes sense since it is a shared flow w/ replay of 1 -  https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines.flow/-state-flow/